server/src/utils.ts

```diff
@@ -803,7 +803,7 @@ export function getCommandLocationOnSystem(
     return null;
   }
 
-  return firstChoice;
+  return [result , ...cmdComponents.slice(1, cmdComponents.length)].join(" ");
```

Comment:
maybe replace result with firstChoice?

I made that change a few weeks ago, `result` returns the absolute path to the executable file. `firstChoice` returns only the executable name. (and if there is a `result` it's because where/which found the command in PATH, so we are sure that we can use it only by its name)

Reply:
Done in f90ea984bac489d11005565667c097be0ab66d5d